ProjetSupaero2018/catkin_ws/src/car_model/scripts/car_model.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env python
+# license removed for brevity
+import rospy
+from std_msgs.msg import String
+
+
+class CarModel:
+    def __init__(self, init_x, init_y):
+        """ Create a car given its initial position
+        """
+        self.x = init_x
+        self.y = init_y
+
+    def set_position(self, dx, dy):
+        """ Change the position of the car given a control value (dx, dy)
+        """
+        self.x += dx
+        self.y += dy
+
+
+if __name__ == '__main__':
+    try:
+        # Create the car model
+        car_model = CarModel(0.0, 0.0)
+
+        publisher = rospy.Publisher('t_car_position', String)
```

Comment:
Missing queue_size : intended ?
Triggering warning when : rosrun car_model car_model.py

Reply:
Good point, I wasn't sure it was required (I expected a default value)